### DISCUSSION TITLE
Traditional Medical Plans without a provider network

### DISCUSSION THREAD

**Discussion by Ginger228** [CMSgov/price-transparency-guide#531] [Q&A] [answered]
@shaselton  @shaselton-usds 
Hi, 
We still have traditional plans - comprehensive, hospital, and major medical.  The products do not have In-Network or Out of Network benefits.  Should these plans be excluded from Transparency in Coverage? 

If there aren't any participating or non-participating providers to list for either the INN or AA MRF. 

Thanks in advance.

**Comment by CKreklow** [comment]
Our interpretation here is that the distinction between in-network and out-of-network is just the presence of a contract agreement, it does not implicate member benefits.  Do you not have a network applied to the plans that allows you to use negotiated rates with some providers?

We still apply networks to these plans, and any provider with a contract in that network goes to the in-network file.  All the claims priced without reference to a contractual agreement are going to the allowed amount file (subject to aggregation and the claim threshold of course).

In the case where we had a plan with 100% U&C pricing, I believe we would intend to report an empty in-network file and put all the data in the allowed amounts file.

**Comment by Ginger228** [comment]
@shaselton @shaselton  Should an empty INN MRF be created for these plans or no INN at all.

**Comment by shaselton-usds** [comment, marked as answer]
@Ginger228 If the plan does not have any contracts with any providers/networks for any items/services and follows [these types of arrangements](https://www.federalregister.gov/documents/2020/11/12/2020-24591/transparency-in-coverage#p-646), then you wouldn't need to produce an in-network file. There would be no information there because you don't have any contracts to disclose.

So, does your plan have contractual in-network provider rates for covered items and services?

**Reply by Ginger228** [reply]
We do not have contractual in-network provider rates for our traditional indemnity product (major medical, hospitalization, or comprehensive).
